pokesets/Manga/Pokemon Adventures/PS-Silver.yaml

```diff
@@ -0,0 +1,148 @@
+species: Weavile
+setname: PS-Silver
+ingamename: Weavile
+displayname: Weavile
+ability: [Pressure]
+item: [null]
+ball: [Poke]
+moves:
+- [Icy Wind]
+- [Blizzard]
+- [Thief]
+- [Assurance]
+gender: [m]
+happiness: 255
+biddable: true
+rarity: 1.0
+nature: Quirky
+evs: {hp: 0, atk: 252, def: 0, spA: 4, spD: 0, spe: 252}
+ivs: 31
+tags: [PS-Silver, manga]
+---
+species: Feraligatr
+setname: PS-Silver
+ingamename: Feraligatr
+displayname: Feraligatr
+ability: [Torrent]
+item: [null]
+ball: [Poke]
+moves:
+- [Ice Punch]
+- [Surf]
+- [Return]
+- [Hydro Cannon]
+gender: [m]
+happiness: 255
+biddable: true
+rarity: 1.0
+nature: Quiet
+evs: {hp: 252, atk: 252, def: 0, spA: 4, spD: 0, spe: 0}
+ivs: 31
+tags: [PS-Silver, manga]
+---
+species: Honchkrow
+setname: PS-Silver
+ingamename: Honchkrow
+displayname: Honchkrow
+ability: [Insomnia]
+item: [null]
+ball: [Poke]
+moves:
+- [Fly]
+- [Wing Attack]
+- [Pursuit]
+- [Peck]
+gender: [m]
+happiness: 255
+biddable: true
+rarity: 1.0
+nature: Adamant
+evs: {hp: 248, atk: 252, def: 0, spA: 0, spD: 8, spe: 0}
+ivs: 31
+tags: [PS-Silver, manga]
+---
+species: Ursaring
+setname: PS-Silver
+ingamename: Ursaring
+displayname: Ursaring
+ability: [Guts]
+item: [null]
+ball: [Ultra] # Heavy Ball
+moves:
+- [Slash]
+- [Fire Punch]
+- [Fury Swipes]
+- [Avalanche]
+gender: [m]
+happiness: 255
+biddable: true
+rarity: 1.0
+nature: null
+evs: {hp: 248, atk: 252, def: 0, spA: 0, spD: 8, spe: 0}
+ivs: 31
+tags: [PS-Silver, manga]
+---
+species: Ursaring
+setname: PS-Silver
+ingamename: Ursaring
+displayname: Ursaring
+ability: [Guts]
+item: [null]
+ball: [Ultra] # Heavy
+moves:
+- [Slash]
+- [Fire Punch]
+- [Fury Swipes]
+- [Avalanche]
+gender: [m]
+happiness: 255
+biddable: true
+rarity: 1.0
+nature: null
+evs: {hp: 248, atk: 252, def: 0, spA: 0, spD: 8, spe: 0}
+ivs: 31
+tags: [PS-Silver, manga]
+---
+species: Gyarados
+setname: PS-Silver
+ingamename: Gyarados
+displayname: Gyarados
+ability: [Intimidate]
+item: [null]
+ball: [Poke]
+shiny: true
```

Comment:
Pretty sure that for shiny Pokemon, you have to make hidden equal false. It's been a while, so when you get permission, you can check via the python verif